### DISCUSSION TITLE
Is it possible to save created form?

### DISCUSSION THREAD

**Discussion by ehsunoo** [GrapesJS/grapesjs#5731] [Q&A]
Is it possible to create a form using existing blocks (such as input fields, select dropdowns, etc.), configure it, save the form data in remote storage (a database), and use it in the future? I understand that while you can save the entire GrapesJS data, i specifically need to export a specified form block and import it into another page, with the form’s structure and configuration fetched from the database.

Thanks.

**Comment by artf** [comment]
Sure, you could save the component as a JSON and then append it later in some other place.

ps. Please be sure to read the documentation before jumping to asking further questions like how to do it.
